src/pghstore/_native.py

```diff
@@ -177,7 +177,9 @@ def key_map(key):
             value = None
         elif not (
             isinstance(value, six.string_types)
-            or isinstance(value, six.binary_type)
+            or isinstance(  # noqa W503 (this is how black formats it)
+                value, six.binary_type
+            )
```

Comment:
Instead of the weird formatting from `black` can't we just combine them?

```python
elif not isinstance(value, (six.string_types, six.binary_type))
```

Reply:
Ah, much better! It's been updated.